### DISCUSSION TITLE
Please Help me

### DISCUSSION THREAD

**Discussion by AboQasem1** [rubenlagus/TelegramBots#972] [Q&A] [answered]
Hello guys
i do download telegrambots 5.3.0 java library
but no exsist "getBotUsername" and "onUpdateReceived" override methods
only getBotToken method.
i use netBeans editor.
please helpme.

https://www.mediafire.com/view/uwh0japqin5gbkw

**Comment by NaveenB2004** [comment, marked as answer]
Hello!

I had the same problem. An **update to the latest version** solved my problem.

Hope this will help you!
